### PR TITLE
feat: Add refresh token to signOut request body

### PR DESCRIPTION
**Pull request by MisterNox**
### 🔗 Linked issue
#730 

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
Adds the refresh token to the signOut request body with the key set according to the defined `refreshRequestTokenPointer`.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have added tests (if possible).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by zoey-kaiser — APPROVED** [review]
Hi @MisterNox 👋 

Thank you so much for your contribution! As we are currently aiming for a larger 0.8 release, we will add your enchantment to that release. 

@phoenix-ru, I think this PR is good to go, I would recommend that after your reviews we can merge this in the next batch and relase it in the next `0.8.0-alpha.2` release!

File: src/runtime/composables/refresh/useAuth.ts
Comment (phoenix-ru) [inline src/runtime/composables/refresh/useAuth.ts:175]: May be better to build body only when the check is truthy:
```ts
const body = method.toLowerCase() === 'get'
  ? undefined
  : objectFromJsonPointer(config.refreshToken.refreshRequestTokenPointer, refreshToken.value)
```

**Review by phoenix-ru — DISMISSED** [review]

**Comment by zoey-kaiser** [comment]
Hi @MisterNox  👋 

Have you had a chance to look at the review yet? Otherwise are there any other open questions I can help you with?

**Comment by sandrinejoy** [comment]
when will this be added to a stable release?

**Review by zoey-kaiser — APPROVED** [review]

File: src/runtime/composables/refresh/useAuth.ts
Comment (zoey-kaiser) [reply src/runtime/composables/refresh/useAuth.ts:177]: @phoenix-ru, as this PR is bascilly ready-to-go and has been stale for a long time, I would go ahead and merge this PR! We can optimize this in a follow-up PR!

**Pull request merged**
